### PR TITLE
Format Zipcode

### PR DESCRIPTION
**Pull request by jtormey**

### CODE REVIEW

**Comment by jtormey** [comment]
Commit message says "6 to 10 digit", actually should be "5 to 9" digit (code is correct though).

**Comment by jtormey** [comment]
Rebased to avoid conflicts, and fixed commit message.

**Comment by plondon** [comment]
If I enter zipcode `10021-1111` I get a 400 from SFOX with the following `{error: 'zipcode is required'}`

**Comment by jtormey** [comment]
Possible that SFOX doesn't accept extended zip codes. Will clarify tomorrow.

**Comment by plondon** [comment]
Also setAddress() saves the unaccepted zipcode to your profile, so even if you change zipcode to an acceptable format the request will fail again.

Should still be fixed on SFOX's end, or at least clarified like you said. Thanks!

**Comment by jtormey** [comment]
Just going to use the standard 5-digit zip code.

File: assets/js/directives/format-input.directive.js [hunk 1]
```diff
@@ -12,21 +12,26 @@ function formatInput () {
 
   function link (scope, elem, attrs, ctrl) {
     let inputFormat = attrs.formatInput;
-    let language = { digit: 'x' };
+    let language = { digit: 'x', open_group: '(', close_group: ')' };
```
Comment (plondon) [inline assets/js/directives/format-input.directive.js]: Are open_group and close_group required still? Is this left over from zipcode 12345-(1211)?

Comment (jtormey) [inline assets/js/directives/format-input.directive.js]: Not required, will remove.

Comment (jtormey) [inline assets/js/directives/format-input.directive.js]: All set.

**Pull request merged**